calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.4-release-notes.mdx

```diff
@@ -0,0 +1,9 @@
+DD March 2024
+
+### Improvements
+-
+
+### Bug fixes
```

Comment:
- If kube-controllers metric port is set to 0, no ingress rule will be created.

Reply:
Has bug fix for:  Kubernetes audit logs for EKS does not work and pods get stuck with this error: eks-log-forwarder-startup time="2024-01-23T17:42:29Z" level=fatal msg="Error setting up elastic client.

Reply:
@vara2504 How about: "Fixed a bug introduced in v3.17.3 that stopped the eks-log-forwarder deployment from starting."

Reply:
Added.